### PR TITLE
Add information about Pension obligation

### PR DESCRIPTION
**Pull request by lukemorton**

### CODE REVIEW

File: benefits/pensions.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<table>
+  <tr>
+    <th>&nbsp;</th>
+    <th>Our (employer) contribution</th>
+    <th>Your (employee) contribution</th>
+  </tr>
+  <tr>
+    <th>Current</th>
+    <td>1%</td>
+    <td>1%</td>
```
Comment (davidwinter) [inline benefits/pensions.md]: Are these the defaults? If so, might be worth stating that. I remember when I completed the form it asked me what I wanted to contribute and didn't mention any default amounts.

Comment (lukemorton) [inline benefits/pensions.md]: What do you mean by defaults?

File: README.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ Benefits & Perks
 * [Friday Lunches & Drinks](benefits/friday_lunch_drinks.md)
 * [Remote Working](benefits/remote_working.md)
 * [Season Ticket Loan](benefits/season_ticket_loan.md)
+* [Pensions](benefits/pensions.md)
```
Comment (RoryMacDonald) [inline README.md]: Benefit or obligation?

File: benefits/pensions.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Pensions
+
+We provide a pension plan which everyone is automatically enrolled on.
+
+You will be enrolled as soon as you pass your probation period which will be stated your employment contract.
```
Comment (sowasred2012) [inline benefits/pensions.md]: Grammar: "...your probation period, which will be stated in your employment contract."

Comment (RoryMacDonald) [inline benefits/pensions.md]: Worth mentioning provider (Royal London?)

File: benefits/pensions.md (hunk 1 above)
Comment (davidwinter) [inline benefits/pensions.md]: When auto-enrolled onto the pension scheme, the default amount you contribute. If that is even a thing? Does everyone have to complete that Royal London form and specify exactly what they want to contribute?

Comment (lukemorton) [inline benefits/pensions.md]: Right gotcha, yes these are the default amounts. As noted further down you can choose to increase if you want. Do you have a suggestion for making it clearer here?

File: README.md (hunk 2 above)
Comment (lukemorton) [inline README.md]: Both probably?

File: benefits/pensions.md (hunk 3 above)
Comment (lukemorton) [inline benefits/pensions.md]: Thanks :)

File: benefits/pensions.md (hunk 1 above)
Comment (davidwinter) [inline benefits/pensions.md]: Maybe just mention it further up?

File: README.md (hunk 2 above)
Comment (RoryMacDonald) [inline README.md]: I think this belongs in Guides, not Benefits.

**Comment by lukemorton** [comment]
@RoryMacDonald Moved to "guides/policies". That better?

**Comment by idabmat** [comment]
Nice @lukemorton . As someone who just came to work in the UK this makes it clearer.

**Pull request merged**